src/worldmap/worldmap_sector.cpp

```diff
@@ -62,14 +62,16 @@ WorldMapSector::current()
 WorldMapSector::WorldMapSector(WorldMap& parent) :
   Base::Sector("worldmap"),
   m_parent(parent),
-  m_camera(new Camera),
+  m_camera(nullptr),
   m_tux(&add<Tux>(&parent)),
   m_spawnpoints(),
   m_initial_fade_tilemap(),
   m_fade_direction()
 {
   BIND_WORLDMAP_SECTOR(*this);
 
+  m_camera.reset(new Camera(*this));
```

Comment:
Can the `new Camera(*this)` part be moved to the initilization list or would that lead to things being uninitializaed?

Reply:
Works indeed, I thought it might make problems, because the object is not fully constructed, yet. But clang seems to be fine with that, and so am I.